### PR TITLE
wallet airdrop: Fetch a new last_id to prevent DuplicateSignature errors during AccountInUse retries

### PR DESCRIPTION
**Pull request by mvines**
Attempts to fix the intermittent CI failure that occurred at https://buildkite.com/solana-labs/solana/builds/6169#ff62509f-f382-43a9-a886-538aff015182

### CODE REVIEW

**Comment by garious** [comment]
Move that to thin_client.rs?

File: src/wallet.rs [hunk 1]
```diff
@@ -783,15 +783,31 @@ fn request_and_confirm_airdrop(
         };
         match status {
             RpcSignatureStatus::AccountInUse => {
-                last_id = get_last_id(rpc_client)?;
+                // Fetch a new last_id, to prevent the retry from getting rejected as a
```
Comment (garious) [inline src/wallet.rs]: Why are you implementing a DuplicateSignature handler in the AccountInUse handler?

**Comment by mvines** [comment]
Bah this is wrong.  `request_airdrop_transaction()` needs the new last_id

File: src/wallet.rs (hunk 1 above)
Comment (mvines) [reply src/wallet.rs]: I'm not.  When `AccountInUse` happens, we need to fetch a new last_id to prevent `DuplicateSignature` on the next retry

Comment (garious) [reply src/wallet.rs]: got it, thanks!

**Comment by mvines** [comment]
> Move that to thin_client.rs?
Can't unfortunately, as `request_airdrop_transaction()`, which invokes the drone, needs a last_id so the drone can sign the transaction itself

**Comment by mvines** [comment]
@garious this PR is fixed up now if you care to take another look.  It's not fancy stuff though

**Pull request merged**